src/index.ts

```diff
@@ -35,6 +35,7 @@ const DEFAULT_OPTIONS: Partial<https.RequestOptions> = {
 
 const sslChecker = (
   host: string,
+  validateSubjectAltName: boolean,
   options: Partial<https.RequestOptions> = {}
```

Comment:
Let's keep `validateSubjectAltName` part of the options instead of having own parameter?

Reply:
@dyaa - Not sure I follow - this is a custom boolean config - Are you saying to change the logic to make the https request itself different depending on that flag? Currently I'm only using the flag post http request to decide whether to validate `subjectaltname` response or not.

Reply:
See latest commit - I've moved the logic into the http request/response itself - does that work?